docs/usage/self-hosted-configuration.md

```diff
@@ -313,19 +313,19 @@ For example, if you set `dockerChildPrefix=myprefix_` then the final container c
 
```

Comment:
wrong merge?

Reply:
@viceice That's because the branch `v35` has been force pushed. I've resolved the conflicts.